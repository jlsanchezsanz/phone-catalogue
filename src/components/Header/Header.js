import React from 'react';

export default function Header() {
  return (
    <div>
      <nav class='navbar navbar-dark bg-dark'>
        <a class='navbar-brand' href='./'>
          Phone Catalogue
        </a>
        <img src={require(`../../assets/cart.svg`)} alt='Cart' />
      </nav>
    </div>
  );
}
