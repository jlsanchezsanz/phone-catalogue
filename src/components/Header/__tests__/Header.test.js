import React from 'react';
import { shallow } from 'enzyme';

import Header from '../Header';

describe('Header', () => {
  it('should display phone gallery', () => {
    const component = shallow(<Header />);
    expect(component).toMatchSnapshot();
  });
});
