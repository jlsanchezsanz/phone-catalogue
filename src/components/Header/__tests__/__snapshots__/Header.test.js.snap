// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should display phone gallery 1`] = `
<div>
  <nav
    class="navbar navbar-dark bg-dark"
  >
    <a
      class="navbar-brand"
      href="./"
    >
      Phone Catalogue
    </a>
    <img
      alt="Cart"
      src={
        Object {
          "ReactComponent": Object {
            "$$typeof": Symbol(react.forward_ref),
            "render": [Function],
          },
          "__esModule": true,
          "default": "cart.svg",
        }
      }
    />
  </nav>
</div>
`;
